examples_test.go

```diff
@@ -19,9 +19,7 @@ func ExampleUnpackedError_ToJSON_external() {
 
 	// unpack and print the error
 	err := readFile("example.json")
-	uerr := eris.Unpack(err)
-	format := eris.NewDefaultFormat(false) // false: omit stack trace
-	u, _ := json.Marshal(uerr.ToJSON(format))
+	u, _ := json.Marshal(eris.ToJSON(err, false)) // false: omit stack trace
```

Comment:
Fix all example links. You need to change their names so they're correctly linked to methods in godoc. e.g. ExampleUnpackedError_ToJSON_external -> ExampleToJSON_external